main.go

```diff
@@ -0,0 +1,186 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/bitrise-io/go-utils/command"
+	"github.com/bitrise-io/go-utils/command/rubycommand"
+	"github.com/bitrise-io/go-utils/errorutil"
+	"github.com/bitrise-io/go-utils/log"
+	"github.com/bitrise-tools/go-steputils/stepconf"
+)
+
+// Config ...
+type Config struct {
+	RepositoryURL string `env:"repository_url,required"`
+
+	GithubAPIToken   stepconf.Secret `env:"github_api_token"`
+	GithubHost       string          `env:"github_host"`
+	GithubAPIBaseURL string          `env:"github_api_base_url"`
+
+	GitlabAPIToken   stepconf.Secret `env:"gitlab_api_token"`
+	GitlabHost       string          `env:"gitlab_host"`
+	GitlabAPIBaseURL string          `env:"gitlab_api_base_url"`
+}
+
+func main() {
+	var cfg Config
+	if err := stepconf.Parse(&cfg); err != nil {
+		failf("Issue with input: %s", err)
+	}
+
+	// Fix repository URL
+	cfg.RepositoryURL = strings.TrimLeft(cfg.RepositoryURL, "https://")
+
+	stepconf.Print(cfg)
+	fmt.Println()
+
+	validateInputs(cfg)
+
+	//
+	// Set local envs for the step
+	{
```

Comment:
This block is unnecessary. All the nested blocks declare their own local variables, thus, there are no variables strictly in this scope.

There are other instances of this later in the code, I won't comment them one-by-one.